pages/osx/rubocop.md

```diff
@@ -0,0 +1,31 @@
+# rubocop
+
+> Lint Ruby files.
+
+- Check all files in the current directory (including subdirectories):
+
+`rubocop`
+
+- Check one or more specific files or directories:
+
+`rubocop /path/to/file /path/to/directory`
```

Comment:
Sorry, regarding @agnivade's point, this should be `{{path/to/file}}` and `{{path/to/directory}}`. 👍